km_cli/km_client.c

```diff
@@ -17,7 +17,7 @@
 /*
  * Command line description:
  *
- * km_cli [-c cmdname] [-p processid] [-d snapshotdir] [-s socket_name] [-l]
+ * km_cli [-c cmdname] [-p processid] [-d snapshotdir] [-s socket_name] [-l] [-t]
```

Comment:
This changes the default behavior. I'd rather we keep it. IOW have an option to continue instead.

Reply:
Only the -s flag implied the payload terminated after the snapshot was completed.
Nothing used km_cli until I started testing it in the bats basic_snapshot test a few weeks ago.
That testing used the new flags I added, nothing used the existing -s flag.
The new flags all assumed that snapshot requests did not terminate the payload.
This PR is the first time we test with the -s flag, so nothing will be affected by this small change in semantics.
I will leave things as they are now in this change.